apps/backend/src/data-marts/use-cases/get-data-mart-run.service.ts

```diff
@@ -0,0 +1,29 @@
+import { Injectable, NotFoundException } from '@nestjs/common';
+import { DataMartRunDto } from '../dto/domain/data-mart-run.dto';
+import { GetDataMartRunCommand } from '../dto/domain/get-data-mart-run.command';
+import { DataMartMapper } from '../mappers/data-mart.mapper';
+import { DataMartRunService } from '../services/data-mart-run.service';
+import { DataMartService } from '../services/data-mart.service';
+
+@Injectable()
+export class GetDataMartRunService {
+  constructor(
+    private readonly dataMartService: DataMartService,
+    private readonly dataMartRunService: DataMartRunService,
+    private readonly mapper: DataMartMapper
+  ) {}
+
+  async run(command: GetDataMartRunCommand): Promise<DataMartRunDto> {
+    await this.dataMartService.getByIdAndProjectId(command.dataMartId, command.projectId);
```

Comment:
Not critical: I like the idea to check projectId and dataMartId alignment, but maybe we can pass projectId into dataMartRunService and use it in `where` condition?  Same goes to `list-runs` use-case.